packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -22,7 +22,7 @@ import { Icon } from '../Icon/Icon';
 
 export type TooltipInteraction = 'none' | 'hover';
 
-export const allowedTooltipFrameProps = ['cursor'] as const satisfies FramePropsKeys[];
+export const allowedTooltipFrameProps = ['cursor', 'margin'] as const satisfies FramePropsKeys[];
```

Comment:
so do we need it? Actually not sure if this margin shouldn't be applied on the area when you hover

Reply:
damn, didn't delete this, thanks